### PR TITLE
19/06/03 dotnet port

### PR DESCRIPTION
**Pull request by daz10000**

### CODE REVIEW

**Comment by chrismacklin** [comment]
Wow, this is a beast.

File: tests/fss.Tests/test_db_mysql.fs [hunk 1]
```diff
@@ -338,39 +369,40 @@ type TestTransactions() = class
...
-
-        trans2.Commit()
-
-        // the conn only sees 1 result, because only trans2 has been committed
-        let results2: test1 [] = conn.Query "select * from test1"  |> Array.ofSeq
-        Assert.IsTrue(results2.Length=1)
-        Assert.IsTrue(conn.ExecuteScalar "SELECT COUNT(*) FROM test1" :?> int64 = 1L)
-        // trans see's 4, because READCOMMIT allows us to see the commit
-        // from trans2 as well as our own.
+    // broken
```
Comment (chrismacklin) [inline tests/fss.Tests/test_db_mysql.fs:377]: Perhaps it is time to just entirely drop support for mysql?  Amyris isn't using it anymore; no idea if there are other clients of this framework.

**Comment by daz10000** [comment]
Yeah,  I'm still doing a tiny bit of work to get the nuget packaging working so this isn't quite ready.  I did get mysql mostly working except for 3 tests so I'll leave it for now but was sorly tempted to put it out of its misery - even setting up a test db is a PITA

**Comment by daz10000** [comment]
This should be good for review now if you're interested.  I need one of the dotnet core friendly libraries, so would like to push that next 24 hours or so.  Ok if you don't have time to review.

**Comment by daz10000** [comment]
BTW, I have struggled to get build.sh working.  Follow readme to just install fake as a dotnet cli tool and just do fake run  long story

**Comment by daz10000** [comment]
Should just build with ./build.sh in root but would be nice to get an independent test of that,

**Comment by chrismacklin** [comment]
Built successfully on my linux box up to tests, but I don't have the requisite databases set up to run the unit tests.

**Pull request merged**